Demo/Demo/SwiftUIComponents/VenmoPayments/VenmoPaymentView.swift

```diff
@@ -0,0 +1,14 @@
+import SwiftUI
+
+struct VenmoPaymentView: View {
+    
+    @ObservedObject var venmoPaymentsViewModel = VenmoPaymentsViewModel()
```

Comment:
My understanding is that @StateObject should be used where the VM is instantiated.